lib/gitlab/client/users.rb

```diff
@@ -127,13 +127,18 @@ def session(email, password)
     #
     # @example
     #   Gitlab.ssh_keys
+    #   Gitlab.ssh_keys(2)
     #
     # @param  [Hash] options A customizable set of options.
     # @option options [Integer] :page The page number.
     # @option options [Integer] :per_page The number of results per page.
     # @return [Array<Gitlab::ObjectifiedHash>]
-    def ssh_keys(options={})
-      get("/user/keys", query: options)
+    def ssh_keys(user_id=nil, options={})
```

Comment:
👍  For making it part of the option.

Changing the method signature is not an option without deprecation.